src/I18n/MessagesFileLoader.php

```diff
@@ -126,7 +125,7 @@ public function __invoke()
         }
 
         if (!$file) {
-            return $package;
+            return false;
```

Comment:
If that's true then I don't know how to deal with non-existing files. It's perfectly fine for a package to be empty.